ClinGen/clincoded
# Connects to #1040 kl external links for no NC values

This PR implements the requirement to change browser display info in static header area, bottom of Basic Info tab and Predictors/In-frame Indel tab for variants without NC_ values (GRCh37 and GRCh38).

Changes are:
 1. Change "UCSC" to "UCSC Browser".
 2. Add links and icons to external source names (UCSC Browser, Variation Viewer and Ensembl Browser) for variants without NC_ values (GRCh37 and GRCh38).

**Test**

* Start VCI, use ClinVar variant 65922 and expect to see links and icons in static header area (green bar) and at bottom of Basic Info tab as:
![screen shot 2016-10-05 at 2 28 52 pm](https://cloud.githubusercontent.com/assets/9206696/19132846/50edd29e-8b0a-11e6-9076-2b9f884f99c1.png)

and in Predictors/In-frame Indel tab as:
![screen shot 2016-10-05 at 2 47 43 pm](https://cloud.githubusercontent.com/assets/9206696/19132936/c358e828-8b0a-11e6-99a1-1c9827e0acc4.png)

* Click link at `UCSC Browser` in static header area (green bar) and expect to open web page "https://genome.ucsc.edu/cgi-bin/hgGateway" in a new browser tab.

* Click link at `Variation Viewer` in static header and expect to open web page "https://www.ncbi.nlm.nih.gov/variation/view/" in a new browser tab.

* Click link at `Ensembl Browser` in static header area and expect to open web page "http://uswest.ensembl.org/Homo_sapiens/Info/Index" in a new browser tab.

* Click the 3 links at the bottom of Basic Info tab and expect to open the same web pages respectively.

* Click the 3 links in Predictors/In-frame Indel tab and expect to get the same result.

**End**
